letsencrypt/cli.py

```diff
@@ -890,7 +892,13 @@ def _create_subparsers(helpful):
     helpful.add_group("revoke", description="Options for revocation of certs")
     helpful.add_group("rollback", description="Options for reverting config changes")
     helpful.add_group("plugins", description="Plugin options")
-
+    helpful.add(
+        "certonly", "--user-agent", type=str, default=None,
```

Comment:
In the meantime I think this PR is sort of okay as is. `letsencrypt --help run` is currently pretty unhelpful, but that's larger than this PR, and we need #1460 to solve it I think.